common.gypi

```diff
@@ -30,10 +30,6 @@
     'openssl_fips%': '',
     'openssl_no_asm%': 0,
 
-    # Some STL containers (e.g. std::vector) do not preserve ABI compatibility
-    # between debug and non-debug mode.
-    'disable_glibcxx_debug': 1,
```

Comment:
So this may not have had the desired effect (being overridden by the value in toolchain.gypi) but I'm a little hesitant about removing it completely from `common.gypi` from the release lines as that's visible to addon authors using `node-gyp`.

Reply:
I know it's not an exact science but GH search doesn't turn up any binding.gyp files nor any JS or TS source files that contain the string disable_glibcxx_debug.

I think it's a pretty safe bet it's unused because it was non-functional anyway.